requirements/base.txt

```diff
@@ -1,66 +1,35 @@
 #
-# This file is autogenerated by pip-compile with python 3.8
+# This file is autogenerated by pip-compile
 # To update, run:
 #
 #    pip-compile --allow-unsafe --no-emit-index-url --output-file=requirements/base.txt requirements/base.ini
 #
-appdirs==1.4.4
-    # via virtualenv
-attrs==20.3.0
-    # via jsonschema
-certifi==2020.12.5
-    # via requests
-chardet==3.0.4
-    # via requests
-distlib==0.3.1
-    # via virtualenv
-filelock==3.0.12
-    # via
-    #   tox
-    #   virtualenv
-idna==2.10
-    # via requests
-jsonschema==3.2.0
-    # via -r requirements/base.ini
-kafka-python==2.0.2
-    # via -r requirements/base.ini
-packaging==20.7
-    # via tox
-pendulum==2.1.2
-    # via -r requirements/base.ini
-pluggy==0.13.1
-    # via tox
-py==1.9.0
-    # via tox
-pyparsing==2.4.7
-    # via packaging
-pyrsistent==0.17.3
-    # via jsonschema
-python-dateutil==2.8.1
-    # via pendulum
-pytzdata==2020.1
-    # via pendulum
-redis==3.5.3
-    # via -r requirements/base.ini
-requests==2.25.1
-    # via -r requirements/base.ini
-six==1.15.0
-    # via
-    #   jsonschema
-    #   python-dateutil
-    #   tox
-    #   virtualenv
-toml==0.10.2
-    # via tox
-tox==3.20.1
-    # via -r requirements/base.ini
-urllib3==1.26.4
-    # via requests
-virtualenv==20.2.2
-    # via tox
-websockets==8.1
-    # via -r requirements/base.ini
+appdirs==1.4.4            # via virtualenv
```

Comment:
I think we should at some point settle on the version of pip-tools when running `fab compile` , in order to get rid of huge diffs like this which are mostly caused by a change in the format of the pip-compile tool. How about updating pip-tools to a version greater than 6.2.0 and try compiling the requirements again ? If it does not do the trick then please do not spend more time on it, it is minor.

Reply:
I agree with Spyros; updating `pip` and `pip-tools` to the latest version should be part of the installations process, but also of the requirements update process. Unfortunately, adding `pip` or `pip-tools` to the requirements files doesn't work and it's also discouraged, so this must be done manually. We can add a paragraph in our docs if it's not already there.

Reply:
@spyrostz @fievelk updated the pip-tools and yes I agree that we should all use the same version